ordered_product/tests/test_ordered_product_page.py

```diff
@@ -0,0 +1,188 @@
+import pytest
+from supplier_product.models import SupplierProduct
+from delivery_location.models import DeliveryLocation
+from ordered_product.models import OrderedProduct
+from product.models import Product
+from django.core.exceptions import ObjectDoesNotExist
+
+
+@pytest.fixture
+def valid_quantity():
+    return '5'
+
+
+@pytest.fixture
+def supplier_product0_id(supplier_product0):
+    return supplier_product0.supplier_product_id
+
+
+@pytest.fixture
+def invalid_supplier_product_id():
+    return '0'
+
+
+@pytest.fixture
+def invalid_delivery_location_id():
+    return '9797'
+
+
+@pytest.fixture
+def client0_login(client, saved_client0):
+    client.force_login(user=saved_client0.client_account)
+
+
+class TestOrderList:
+    @pytest.mark.django_db()
+    def test_orders_list_no_conatins_orders(self, client, saved_client0, client0_login):
+        response = client.get('/ordered_product/orders')
+        assert response.status_code == 200
+        template_names = set(template.origin.template_name for template in response.templates)
+        assert 'ordered_product/orders.html' in template_names
+        clients_orders_by_the_web_view = list(response.context['orders'])
+        client_orders_by_objects_filter = list(OrderedProduct.filter_user_name(saved_client0))
+        assert clients_orders_by_the_web_view == []
+        assert client_orders_by_objects_filter == []
+        assert response.context['message'] == "No orders found"
+
+    @pytest.mark.django_db()
+    def test_orders_list_conatins_orders(self, client, saved_client0, ordered_product0, client0_login):
+        assert ordered_product0.user_name == saved_client0
+        ordered_product0.save_ordered_product()
+        response = client.get('/ordered_product/orders')
+        assert response.status_code == 200
+        template_names = set(template.origin.template_name for template in response.templates)
+        assert 'ordered_product/orders.html' in template_names
+        client_orders_by_the_web_view = list(response.context['orders'])
+        client_orders_by_objects_filter = list(OrderedProduct.filter_user_name(saved_client0))
+        assert client_orders_by_the_web_view == client_orders_by_objects_filter
```

Comment:
> It will be way more readable if you'll separate this to 3 lines: one defining the variable on the right one defining the variable on the left and last one for the assertions.
> 
> make sure to name the variables well, so it'll be clear what you're testing, even if the reviewer does not know Django or HTML

See Omer review ^^

Reply:
You are right, I didn't notice it.